src/clients/python/experimental_api_v2/examples/simple_http_v2_string_infer_client.py

```diff
@@ -74,10 +74,10 @@
     input1_data = in1n.reshape(in1.shape)
 
     # Initialize the data
-    inputs[0].set_data_from_numpy(input0_data, binary_data=False)
+    inputs[0].set_data_from_numpy(input0_data, binary_data=True)
```

Comment:
Should use the identity model to verify that strings that contain special characters and '\0' are handled correctly.

Reply:
Done.